v2/pipe_net.go

```diff
@@ -0,0 +1,34 @@
+package plugin
+
+import (
+	"context"
+	"crypto/tls"
+	"net"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+func newTCPListener() (net.Listener, error) {
+	listener, err := net.Listen("tcp", "[::1]:0")
```

Comment:
Do we need this? Aren't unix sockets enough?

Reply:
Ahh yeah, windows, forgot about this (:. I'm okay with supporting this, tho we maybe wait until people want this feature, I can't imaging that many host their gotify server on windows. Even tho we have many windows exe downloads https://somsubhra.github.io/github-release-stats/?username=gotify&repository=server&page=1&per_page=5